components/ReviewCard.tsx

```diff
@@ -67,19 +83,26 @@ export const ReviewCard: React.FC<ReviewCardProps> = ({ submissionData }) => {
       />
     )
   }
+
   return (
     <>
       {diff && (
-        <div className="card shadow-sm border-0 mt-3 p-3">
+        <div className="card shadow-sm border-0 mt-3">
           <div className="card-header bg-white">
-            <strong>{submissionData.user.username}</strong> submitted Submission
-            ID: {submissionData.id}
+            <Text color="darkgrey" bold>
```

Comment:
I know we didn't have it previously, but this seems to be the title of the review card so it should be wrapped in a heading tag.

Reply:
Right, I didn't want to add too many components off the start. So, for now, I just have the Text component. We can always add additional ones later as needed :)